boards/adafruit-itsy-bitsy-rp2040/examples/adafruit_itsy_bitsy_rainbow.rs

```diff
@@ -4,7 +4,7 @@
 
 use adafruit_itsy_bitsy_rp2040::entry;
 use core::iter::once;
-use embedded_hal::timer::CountDown;
+use embedded_hal_0_2::timer::CountDown;
```

Comment:
Instead of keeping to use `CountDown` from the old version of embedded-hal, it would be better to migrate to the `DelayNs` implementation on `Timer`.

Reply:
Thanks, I'll prepare the changes

Reply:
I was able to replace usage of `CountDown` with `DelayNs`